src/source/PeerConnection/Rtcp.c

```diff
@@ -73,6 +102,15 @@ STATUS onRtcpPacket(PKvsPeerConnection pKvsPeerConnection, PBYTE pBuff, UINT32 b
                         rttPropDelayMsec = KVS_CONVERT_TIMESCALE(rttPropDelay, DLSR_TIMESCALE, 1000);
                         DLOGD("RTCP_PACKET_TYPE_RECEIVER_REPORT rttPropDelay %u msec", rttPropDelayMsec);
                     }
+
+                    MUTEX_LOCK(pTransceiver->sender.statsLock);
+                    pTransceiver->sender.remoteInboundRtpStreamStats.reportsReceived++;
+                    pTransceiver->sender.remoteInboundRtpStreamStats.fractionLost = (DOUBLE) fractionLost / 255.0;
+                    pTransceiver->sender.remoteInboundRtpStreamStats.roundTripTimeMeasurements++;
+                    pTransceiver->sender.remoteInboundRtpStreamStats.totalRoundTripTime += rttPropDelayMsec;
+                    pTransceiver->sender.remoteInboundRtpStreamStats.roundTripTime = rttPropDelayMsec;
```

Comment:
How would the customer pull this information? I am assuming they will be making periodic calls to `getRtpOutboundStats` in Metrics.c?

Reply:
RtcOutboundRtpStreamStats as well as RtcRemoteInboundRtpStreamStats are transceiver specific
@disa6302 you want to modify `Metrics.c` api ?

Reply:
Sorry. Just having a hard time visualizing. I understand this is transceiver specific. But, without having these exposed to the user through an API/callback user can register to, it would be difficult. Can you may be explain how a customer could consume this information?

Reply:
`STATUS getRtpOutboundStats(PRtcPeerConnection pRtcPeerConnection, PRtcRtpReceiver transceiver, PRtcOutboundRtpStreamStats pRtcOutboundRtpStreamStats)`